frontend/src/App.tsx

```diff
@@ -0,0 +1,38 @@
+import ProductList from "./components/ProductList";
+import ProductView from "./components/ProductView";
+import Cart from "./components/Cart";
+import { ThemeProvider } from "@material-ui/core/styles";
+import { CssBaseline, Button } from "@material-ui/core/";
+import appTheme from "./styles/AppTheme";
+import { useHistory } from "react-router-dom";
+import { Switch, Route, Redirect } from "react-router-dom";
+
+function App() {
+  let history = useHistory();
+  return (
+    <ThemeProvider theme={appTheme}>
+      <CssBaseline>
+        <Button
+          variant="outlined"
+          style={{ margin: "20px" }}
+          color="primary"
+          onClick={() => {
+            history.push("/products");
+          }}
+        >
+          Home
+        </Button>
+        <Switch>
```

Comment:
What if the user is directly accessing a path that you don't have in your switch? For example http://localhost:4200/asdfg. For such cases you need to have a wildcard route that catches all the paths that do not fit with any other declared route and return a 404 page.